### PR TITLE
Stories

### PR DESCRIPTION
**Pull request by Dielone**

### CODE REVIEW

File: app/controllers/stories_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class StoriesController < ApplicationController
+  def index
+    @stories = Story.all
```
Comment (vladimirbazhanov) [inline app/controllers/stories_controller.rb]: This code is fine, but pls add a task to your tracker and name it "Use gem decent_exposure"

Pls read that gem documentation and you will see a magic :)

File: app/views/stories/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<% @stories.each  do |story| %>
+<%= story.name %>
```
Comment (vladimirbazhanov) [inline app/views/stories/index.html.erb]: Please use SLIM markup language for views instead of erb

File: config/database.yml [hunk 3]
```diff
@@ -6,11 +6,11 @@ defaults: &defaults
   pool: <%= [ENV.fetch("MAX_THREADS", 5), ENV.fetch("DB_POOL", 5)].max %>
 
 development:
-  database: <%= ENV.fetch("DATABASE_NAME", File.basename(Rails.root)) %>_dev
+  database: <%= ENV.fetch("penstories_db", File.basename(Rails.root)) %>_dev
```
Comment (vladimirbazhanov) [inline config/database.yml]: database: penstories_db

it will be all you need)

File: db/migrate/20160415112754_create_stories.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+class CreateStories < ActiveRecord::Migration
+  def change
+    create_table :stories do |t|
+      t.integer :user_id
+      t.string :name
+      t.string :body
+      t.timestamps
+      t.belongs_to :user, index: true
```
Comment (vladimirbazhanov) [inline db/migrate/20160415112754_create_stories.rb]: t.belongs_to :user, index: true
and
t.integer :user_id

is the same but first is not working, pls remove it

File: db/migrate/20160415144737_add_role_to_users.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :role, :string
```
Comment (vladimirbazhanov) [inline db/migrate/20160415144737_add_role_to_users.rb]: always add 'default' value for columns, in this case you can use "default: 'reader'" for example

**Comment by vladimirbazhanov** [comment]
:ok: merge it after comments are resolved

File: db/migrate/20160415144737_add_role_to_users.rb (hunk 5 above)
Comment (Dielone) [inline db/migrate/20160415144737_add_role_to_users.rb]: @vladimirbazhanov ok, I'm going to fix these comments, thank you.

**Pull request closed without merging**